README.md

```diff
@@ -30,8 +30,23 @@ SelfCare Header component
 | Prop | Type | Mandatory | Description |
 |------|------|-----------|-------------|
 | withSecondHeader | Y | boolean | if true, it will render an other toolbar under the Header |
+| productsList| Y |Array`ProductEntity`| If withSecondHeader is true, this component will be rendered at the end of the secondary toolbar |
+| selectedPartyId | N | the party selected |
+| selectedProductId | N | the product id selected |
+| partyList | N | Array`PartyEntity` | the parties list |
+| loggedUser | Y | `JwtUser | false` | shows if there is a logged user |
+| assistanceEmail | N | string | the email to which the assistance button will ask to send an email |

```

Comment:
append to the description "if the user is not logged in, otherwise it will be redirect to the assistance form"